srcpkgs/swapspace/files/swapspace/run

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
```

Comment:
why not 
```suggestion
#!/bin/sh -e
```
?

Reply:
`set -e` is slightly more common in our runit scripts, and I personally prefer to keep the flags out of the shebang

Reply:
shebangs can technically only be one argument, but some programs interpret the rest of the line. I think it's better as `set`

Reply:
Also when you run the script directly by passing the file to sh, the shebang line will be ignored.